src/css/reset.css

```diff
@@ -1 +1,41 @@
-/* Base styles */
+body {
+  font-family: 'Comic sans MS', cursive, sans-serif;
+  background-color: #fff;
+  overflow-x: hidden;
+}
+* {
+  margin: 0;
+  padding: 0;
+  border: 0 none;
+  list-style: none outside none;
+  outline: 0 none;
+  box-sizing: border-box;
+}
+button,
+input {
+  overflow: visible;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+a:hover {
+  color: inherit;
+  text-decoration: none;
+}
+
+button, input, optgroup, select, textarea {
+    font-family: inherit;
+    font-size: inherit;
+    line-height: inherit;
+}
+input:focus, select:focus, .btn:focus {
+    border-color: #00ffff;
+  }
+
+  .btn:hover {
+    opacity: 0.9;
+  }
+  
```

Comment:
Can you format this file? This is ignored by styelint but not properly formatted.